lib/butter.ts

```diff
@@ -0,0 +1,52 @@
+import type {
+  AuthorMethods,
+  CategoryMethods,
+  FeedMethods,
+  TagMethods,
+  PageMethods,
+  PostMethods,
+  ContentMethods
+} from './butter.d'
+
+import type { GlobalButterConfig } from './typescript/GlobalButterConfig'
+
+// wrapper for API calls
+import { APIWrapper } from './utilities/apiWrapper'
```

Comment:
I've seen other files use `import type` for APIWrapper, could this be `import { type APIWrapper } ...`?

Also as a note, since the `/utilities/apiWrapper` file could export more than just types, it's best to do `import { type APIWrapper }` as opposed to `import type { APIWrapper }` to slightly better future-proof DX (noticed this on other files).

Reply:
This is specifically importing the class from `apiWrapper.ts`, other places are importing the class as a type (where the actual class is passed into via `constructor()`)

Reply:
Ah gotcha!